app/models/reports/result.rb

```diff
@@ -148,6 +150,15 @@ def count_adjusted_registrations
       end
     end
 
+    def count_cumulative_assigned_patients
+      self.cumulative_assigned_patients = full_data_range.each_with_object(Hash.new(0)) { |period, running_totals|
+        previous_registrations = running_totals[period.previous]
+        current_registrations = assigned_patients[period]
+        total = current_registrations + previous_registrations
```

Comment:
Can you rename these local variables? This isn't counting registrations...maybe just name it `running_totals` and `current_count` for example.